components/Modals/DashboardModal/index.tsx

```diff
@@ -283,15 +284,19 @@ function Component() {
                                     `https://v2.viewblock.io/zilliqa/tx/${deploy[0].ID}?network=${net}`
                                 )
                             }, 1000)
-                            let new_ssi = deploy[0].ContractAddress
-                            new_ssi = zcrypto.toChecksumAddress(new_ssi)
+                            let api = 'https://api.zilliqa.com'
+                            if (net === 'mainnet') {
+                                api = 'https://dev-api.zilliqa.com'
```

Comment:
Hey @devwistkey, it's the other way around, dev for testnet

Reply:
sorry, fixed